packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import useStyles, { StyleSheet } from '@airbnb/lunar/lib/hooks/useStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text, { TextProps } from '../Text';
+
+export const styleSheet: StyleSheet = () => ({
+  dd: {
+    marginInlineStart: 0,
+  },
+});
+
+export type Props = TextProps & {
+  /** Detail links to go after Term label. */
+  after?: React.ReactNode;
+  /** Term content describing the value displayed. */
+  children: NonNullable<React.ReactNode>;
+  /** If enabled, detail links at aligned to the right end. */
+  endAlign?: boolean;
+  /** Term label describing the value displayed. */
+  label: string | React.ReactNode;
+  /** If enabled, term label is uppercased. */
+  uppercased?: boolean;
```

Comment:
Can you add `styleSheet` like all the other components.

Reply:
Got it